### ISSUE TITLE
Why so many anchor boxes??

### ISSUE THREAD

**Issue by qtw1998** [zylo117/Yet-Another-EfficientDet-Pytorch#248]
![img_inferred_d1_this_repo_0](https://user-images.githubusercontent.com/34357479/80913203-6b5a6e00-8d75-11ea-87f4-f8a02e79ff88.jpg)

**Comment by qtw1998** [comment]
And from head no using pretrained pth 
![img_inferred_d1_this_repo_0](https://user-images.githubusercontent.com/34357479/80913365-a27d4f00-8d76-11ea-8dba-e8c083c3ff3f.jpg)

**Comment by qtw1998** [comment]
![img_inferred_d1_this_repo_0](https://user-images.githubusercontent.com/34357479/80916944-6c4bc980-8d8e-11ea-93f6-c9dd9366930a.jpg)
There must be something wrong....

**Comment by zylo117** [comment]
please provide more info

**Comment by qtw1998** [comment]
> please provide more info and the minimal code to replicate this issue
It's the AP values below:
![image](https://user-images.githubusercontent.com/34357479/80931423-52d96a80-8dec-11ea-84eb-8e99f72f9546.png)
It's the loss situation:
![image](https://user-images.githubusercontent.com/34357479/80931445-6f75a280-8dec-11ea-8d02-292e4ac580ad.png)
And it's the latest model inferring result:

**Comment by qtw1998** [comment]
> please provide more info and the minimal code to replicate this issue


![output](https://user-images.githubusercontent.com/34357479/80931486-b499d480-8dec-11ea-8556-98de6332c372.jpg)

**Comment by cendelian** [comment]
what's your learning rate?
I meet this before and I solve it by changed learning rate to 1e-4.
if learning late is too low. it need so more epochs

**Comment by zhaozilongwhu** [comment]
I got the same problem!
